rdflib/plugins/stores/sparqlstore.py

```diff
@@ -425,27 +425,27 @@ class SPARQLUpdateStore(SPARQLStore):
     # in order to avoid unbalanced curly braces.
 
     # From the SPARQL grammar
-    STRING_LITERAL1 = u"'([^'\\\\]|\\\\.)*'"
-    STRING_LITERAL2 = u'"([^"\\\\]|\\\\.)*"'
-    STRING_LITERAL_LONG1 = u"'''(('|'')?([^'\\\\]|\\\\.))*'''"
-    STRING_LITERAL_LONG2 = u'"""(("|"")?([^"\\\\]|\\\\.))*"""'
-    String = u"(%s)|(%s)|(%s)|(%s)" % (
-        STRING_LITERAL1,
+    STRING_LITERAL1 = "'([^'\\\\]|\\\\.)*'"
+    STRING_LITERAL2 = '"([^"\\\\]|\\\\.)*"'
+    STRING_LITERAL_LONG1 = "'''(('|'')?([^'\\\\]|\\\\.))*'''"
+    STRING_LITERAL_LONG2 = '"""(("|"")?([^"\\\\]|\\\\.))*"""'
+    String = '(%s)|(%s)|(%s)|(%s)' % (
+        STRING_LITERAL1, 
         STRING_LITERAL2,
-        STRING_LITERAL_LONG1,
-        STRING_LITERAL_LONG2,
+        STRING_LITERAL_LONG1, 
+        STRING_LITERAL_LONG2
     )
-    IRIREF = u'<([^<>"{}|^`\\]\\\\\[\\x00-\\x20])*>'
-    COMMENT = u"#[^\\x0D\\x0A]*([\\x0D\\x0A]|\\Z)"
+    IRIREF = '<([^<>"{}|^`\\]\\\\[\\x00-\\x20])*>'
+    COMMENT = '#[^\\x0D\\x0A]*([\\x0D\\x0A]|\\Z)'
```

Comment:
This is always a headache to find the correct amount of backslashes. Is it correct to remove one? Is this due to some 2to3 change? do we have some way to express this regex in a more clear representation?